oodeel/extractor/torch_feature_extractor.py

```diff
@@ -224,9 +231,15 @@ def predict(
 
         if isinstance(dataset, get_args(ItemType)):
             tensor = TorchDataHandler.get_input_from_dataset_item(dataset)
-            return self.predict_tensor(tensor, detach=detach)
+            features = self.predict_tensor(tensor, detach=detach)
+            if return_labels:
```

Comment:
In that case, we should be able to also return the label found in the given item, if there is any. Should be features, None if `isinstance(dataset, tf.Tensor)` and features, label otherwise. The problem is: how to identify label in dataset item? dataset[-1]? dataset['label']? (Maybe [1] to be consistent with the rest of your code)